user_tools/docs/qualx.md

```diff
@@ -152,10 +152,13 @@ Once satisfied with the model, just supply the path to this model in the `--cust
 To continually train and evaluate a custom XGBoost model on eventlogs collected over time, run the following periodically:
 ```bash
 spark_rapids train_and_evaluate \
---config qualx-pipeline-conf.yaml
+--qualx_pipeline_config qualx-pipeline-conf.yaml
 --ouptut_folder train_and_evaluate_output
 ```
 
+Arguments:
+- qualx_pipeline_config: (OPTIONAL) Path to YAML config file containing the required training parameters.
+
 #### Configuration
 The training and evaluation pipeline is configured via the `qualx-pipeline-conf.yaml` file, which should have the following format:
 ```yaml
```

Comment:
@leewyang Do you think it's necessary to copy this yaml content into a file under resources and use it as an example in the `train_and_evaluate` `qualx_pipeline_config` argument description?

Reply:
We could.  The main issue is that the `qualx_pipeline_config` is specific to a model/customer, e.g. eventlog locations, dataset names, etc, so it'd only be an example file (not a functional default).